plugin-repos/starlark-apps/tronbyte_repository.py

```
@@ -0,0 +1,472 @@
+
+        content = self._fetch_raw_file(star_path)
+        if not content:
+            return False, f"Failed to download .star file for {app_id} (tried {star_filename})"
+
+        try:
+            output_path.parent.mkdir(parents=True, exist_ok=True)
+            with open(output_path, 'w', encoding='utf-8') as f:
+                f.write(content)
+
+            logger.info(f"Downloaded {app_id}.star to {output_path}")
+            return True, None
```

Ruff (0.15.1)

[warning] Consider moving this statement to an `else` block

(TRY300)